migrate_test.go

```diff
@@ -233,33 +233,33 @@ func TestMigrate(t *testing.T) {
 		expectSeq     migrationSequence
 	}{
 		// migrate all the way Up in single steps
-		{version: 0, expectErr: os.ErrNotExist},
-		{version: 1, expectErr: nil, expectVersion: 1, expectSeq: seq.add(M(1))},
-		{version: 2, expectErr: os.ErrNotExist},
-		{version: 3, expectErr: nil, expectVersion: 3, expectSeq: seq.add(M(3))},
-		{version: 4, expectErr: nil, expectVersion: 4, expectSeq: seq.add(M(4))},
+		{version: 0, expectErr: os.ErrNotExist, expectSeq: seq.add()},
```

Comment:
Wow, I didn't realize we had such unreadable tests...
It's much cleaner to explicitly state/list the expected values instead of "building" them.

Do you mind cleaning this up?